modules/installation-vsphere-installer-infra-requirements.adoc

```diff
@@ -85,6 +85,10 @@ Available resources vary between clusters. The number of possible clusters withi
 == Networking requirements
 
 You must use DHCP for the network and ensure that the DHCP server is configured to provide persistent IP addresses to the cluster machines. Additionally, you must create the following networking resources before you install the {product-title} cluster:
+[discrete]
+[id="installation-vsphere-installer-infra-requirements-networking-NTP_{context}"]
+=== Network Time Protocol (NTP)
+Each OpenShift Container Platform node in the cluster must have access to an NTP server discoverable via DHCP.
```

Comment:
* Openshift Container Platform -> {product-title}
* suggest changing "to an NTP server discoverable via DHCP" -> "to an NTP server that is discoverable via DHCP", for clarity

Reply:
@bergerhoffer ooo yes, applies to 4.5+ and ok pushed those comments :)